dotnet/packages/Microsoft.TeamsAI/README.md

```diff
@@ -0,0 +1,101 @@
+# Teams AI Library
+
+Welcome to the Teams AI Library .NET package! 
+
+This SDK is specifically designed to assist you in creating bots capable of interacting with Teams and Microsoft 365 applications. It is constructed using the [Bot Framework SDK](https://github.com/microsoft/botbuilder-dotnet) as its foundation, simplifying the process of developing bots that interact with Teams' artificial intelligence capabilities. See the [Teams AI repo README.md](https://github.com/microsoft/teams-ai), for general information, and JavaScript support is available via the [js](https://github.com/microsoft/teams-ai/tree/main/js) folder.
+
+Requirements:
+
+*   [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+*   [Azure OpenAI](https://azure.microsoft.com/en-us/products/ai-services/openai-service) resource or an account with [OpenAI](https://platform.openai.com/)
+
+## Getting Started: Migration vs New Project
+
+If you're migrating an existing project, switching to add on the Teams AI Library layer is quick and simple. For a more-detailed walkthrough, see the [migration guide](https://github.com/microsoft/teams-ai/blob/main/getting-started/dotnet/00.MIGRATION.md). The basics are listed below.
+
+### Migration
+### *IMPORTANT Migration code & steps below needs to be updated for C#
+
+In your existing Teams bot, you'll need to add the Teams AI Library SDK package and import it into your bot code.
+
+```bash
+yarn add @microsoft/teams-ai
```

Comment:
since this readme is the `C#` version, I think we should update this readme contents to not have `JS` instructions

Reply:
Hi @aacebo, 

This is where I would need help to make sure there is the right C# code is in place of the JS code specifically in the Migration and Prompt Manager Sections. 

The JS README has this initial info and code with links to more detailed information. Currently, I have the C# README set up similarly, but would need the updated equivalent C# code. Alternatively, I can remove the code in this README and just have the links to the more detailed guides. 

Would you or @singhk97 be the best person to coordinate with on getting the right C# code here?

Reply:
updated